alf/environments/suite_carla.py

```diff
@@ -1187,6 +1188,35 @@ def _reset(self):
         return self._get_current_time_step()
 
 
+@gin.configurable
+class CarlaSingleRewardWrapper(AlfEnvironmentBaseWrapper):
```

Comment:
Suggest to change it to "ScalarRewardWrapper", which accept a weight vector and output the weighted reward.

Reply:
updated